homeassistant/components/tensorflow/image_processing.py

```diff
@@ -294,7 +346,7 @@ def process_image(self, image):
             except UnidentifiedImageError:
                 _LOGGER.warning("Unable to process image, bad data")
                 return
-            img.thumbnail((460, 460), Image.ANTIALIAS)
+            # img.thumbnail((460, 460), Image.ANTIALIAS)
```

Comment:
stale code?

Reply:
Commented for testing. Will uncomment. A side issue should be raised that the OpenCV build in the Home Assistant docker image doesn't include python bindings though.